lib/user_agent/browsers/wondery_crawler.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    # wondery/develop
+    # wondery/development
+    # wondery/prod
+    # wondery/production
+    # wondery/stage
+    # wondery/staging
+    class WonderyCrawler < Base
```

Comment:
Need circle back with Kit on this, initial plan was include this into existing class, then we decided to keep separate for a couple of reasons. Let me check with him again.

Reply:
I'd keep them separated, easier for attributes like `bot?`, `mobile?`, `platform`...

Reply:
bot? = version.match?(WONDERY_CRAWLER_REGEX)
mobile? = !bot?
platform = return if bot?
browser = bot? ? WONDERY_CRAWLER : WONDERY

alternatively:
WONDERY_CRAWLER_VERSIONS = Set.new(%w[develop development stage staging prod production]).freeze
bot? = WONDERY_CRAWLER_VERSIONS.include?(version)

I think they should be together because we have a lot of browsers in the list of things we loop through already and we're doing the same checks against the browser (looking for Wondery) which we will do every single time unless we found a browser match before Wondery